packages/bundler/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@biconomy/bundler",
-  "version": "4.0.1",
+  "version": "4.0.2",
```

Comment:
Latest version of the bundler package is 4.0.0 but here we bump it to 4.0.2, why not 4.0.1 ?